include/envoy/server/overload_manager.h

```diff
@@ -39,13 +40,23 @@ class OverloadActionState {
  */
 using OverloadActionCb = std::function<void(OverloadActionState)>;
 
+enum class OverloadTimerType {
+  // Timers created with this type will never be scaled. This should only be used for testing.
+  UnscaledRealTimer,
+  // The amount of time an HTTP connection to a downstream client can remain idle (no streams).
+  HttpDownstreamIdleConnectionTimeout,
```

Comment:
Consider adding a reference in this comment to the real parameter associated with this enum value.